hadoop-project/pom.xml

```diff
@@ -2414,6 +2415,7 @@
         <configuration>
           <reuseForks>false</reuseForks>
           <forkedProcessTimeoutInSeconds>${surefire.fork.timeout}</forkedProcessTimeoutInSeconds>
+          <surefire.module.config>${surefire.module.config}</surefire.module.config>
```

Comment:
add the options directly:
```xml
<failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>
<trimStackTrace>false</trimStackTrace>
```

see [<failIfNoSpecifiedTests>](https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#failIfNoSpecifiedTests)

Reply:
I had a PR long back doing that, can extract it from there
https://github.com/apache/hadoop/pull/5567/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R126-R128

Reply:
@steveloughran @ayushtkn Thanks for the suggestions! I will continue to improve this pr.